### PR TITLE
test(ccip): add test cases for ccip apis 

### PR DESCRIPTION
**Pull request by erik1110**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Other (please describe)**
testing for attendee #1143 

## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: src/ccip/tests/api/test_ccip.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    data = response.json()
+
+    assert_data_structure(data, 'session_types')
+    assert_data_structure(data, 'tags')
+    assert_data_structure(data, 'rooms')
+    assert_data_structure(data, 'speakers')
+
+
+@pytest.mark.django_db
+def test_session(client):
```
Comment (josix) [inline src/ccip/tests/api/test_ccip.py:34]: **suggestion:** this test seems to test the data schema too, which has the same purpose as above test. Do you consider putting them in the same test function and validating the schema according to the different entities? Probably we could also clarify the purpose of this test by naming it as `test_session_schema`

**Review by josix — CHANGES_REQUESTED** [review]

**Comment by josix** [comment]
hi @erik1110, per discussion, please add the related entities like customEvents, and proposals to the fixtures so that the tests could simulate the actual scenario.

File: src/ccip/tests/api/test_ccip.py (hunk 1 above)
Comment (erik1110) [reply src/ccip/tests/api/test_ccip.py]: OK

**Comment by erik1110** [comment]
> hi @erik1110, per discussion, please add the related entities like customEvents, and proposals to the fixtures so that the tests could simulate the actual scenario.



> hi @erik1110, per discussion, please add the related entities like customEvents, and proposals to the fixtures so that the tests could simulate the actual scenario.

OK, I will add another pull request!

**Review by josix — APPROVED** [review]

**Pull request merged**